lib/course_planner/events/events.ex

```diff
@@ -17,15 +17,30 @@ defmodule CoursePlanner.Events do
 
   def all do
     query = from e in Event,
-    order_by: [desc: e.starting_time, desc: e.finishing_time]
+    order_by: [asc: e.date, asc: e.starting_time, asc: e.finishing_time]
 
     Repo.all(query)
   end
 
+  def all_splitted(now) do
```

Comment:
is there any logic to distinguish between all and the events for a specific person ?
I think again like courses in the index page we should not show other events to people who are not added as participants, but rather provide the option via calendar using "my_classes" option

Reply:
No, it's not. It was created as a default index and I understood that all the events would be visible, but I guess I was wrong.